ArduPlane/Plane.h

```diff
@@ -785,6 +785,9 @@ class Plane : public AP_Vehicle {
     // soaring mode-change timer
     uint32_t soaring_mode_timer;
 
+    // air mode is 0 = not-configured; 1 = disabled; 2 = enabled
```

Comment:
I think this comment is wrong based on the implementation?

Reply:
right, I changed from DISABLED/NONE/ENABLED to off/on due to collisions with macros and forgot to update this